modules/apps/headless/headless-admin-user/headless-admin-user-impl/rest-openapi.yaml

```diff
@@ -1009,7 +1009,6 @@ components:
                     # @review
                     description:
                         The optional external key of this user account.
-                    readOnly: true
```

Comment:
What about the post and put method? Because it's not readonly anymore can we set the ERC also in these methods? In these cases it should not use the previous value if null

Reply:
Thinking about the customers that are using this endpoint and expecting the ERC remains the same, I think that we can just follow the same approach as the PATCH method for this field

Reply:
I think that in POST we can use the value the customer is passing (or leave it empty so it will be autogenerated). In PATCH and in PUT we should keep the "old value" if not passed to avoid breaking changes

Reply:
When I tried to update ERC for the new user I got the same error https://github.com/liferay-commerce/liferay-portal/pull/4627#discussion_r1463382175

Reply:
Ok, we need also in the post but we will manage it in another ticket